config/cron_udp.hjson

```diff
@@ -0,0 +1,364 @@
+## UDP (Unizin Data Platform) queries for MyLA cron job
+"CRON_QUERIES": {
+    "user" :
+        '''
+        select
+            (17700000000000000 + cast(p2.lms_ext_id as bigint)) as user_id,
```

Comment:
This won't work for other schools if we have this 17700000000000000 value hardcoded. This is also used in the submission query. 

They could customize this file but generally we've tried to use the config value `CANVAS_DATA_ID_INCREMENT`. I'd pass this into here.

Reply:
Done. Replaced by settings.CANVAS_DATA_ID_INCREMENT